components/spider-storage/src/bin/grpc_server.rs

```
@@ -0,0 +1,170 @@
+#[derive(Debug, Parser)]
+#[command(about = "Run the Spider storage gRPC server.")]
+struct Cli {
+    /// Address the gRPC server listens on.
+    #[arg(
+        long,
+        env = "SPIDER_STORAGE_LISTEN_ADDR",
+        default_value = "127.0.0.1:50051"
+    )]
+    listen_addr: SocketAddr,
+
+    /// MariaDB host.
```

Clippy (1.96.0)

[error] item in documentation is missing backticks

(error)